go.mod

```diff
@@ -11,8 +11,10 @@ replace github.com/golang/lint => golang.org/x/lint v0.0.0-20190301231843-5614ed
 replace (
 	cloud.google.com/go/pubsub => cloud.google.com/go/pubsub v1.3.1
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.2.0+incompatible
+	github.com/GoogleCloudPlatform/testgrid => github.com/GoogleCloudPlatform/testgrid v0.0.13
 	github.com/docker/docker => github.com/docker/docker v1.4.2-0.20200203170920-46ec8731fbce
 	golang.org/x/lint => golang.org/x/lint v0.0.0-20190409202823-959b441ac422
+	gopkg.in/yaml.v3 => gopkg.in/yaml.v3 v3.0.0-20190709130402-674ba3eaed22
```

Comment:
Why are you forcing a particular yaml version?

Reply:
The `TestGenYAML` tests are failing when upgrading.

Reply:
Please try and fix the tests first

Reply:
Sure, updating the dependency and fixing the tests there: https://github.com/kubernetes/test-infra/pull/19816

Reply:
Done :+1: